lib/compiler-cranelift/src/translator/code_translator.rs

```diff
@@ -1460,6 +1566,11 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             let bool_result = builder.ins().vall_true(a);
             state.push1(builder.ins().bint(I32, bool_result))
         }
+        Operator::I8x16Bitmask | Operator::I16x8Bitmask | Operator::I32x4Bitmask => {
+            unimplemented!("SIMD Operator {:?} not yet implemented", op);
+            // let a = pop1_with_bitcast(state, type_of(op), builder);
```

Comment:
commented-out code

Reply:
It will be useful for the future (is directly ported from cranelift-wasm, but can't be used just yet since it requires for Cranelift to publish a new version), that's why is useful keeping it!